src/components/card-detail-list/index.tsx

```diff
@@ -25,161 +28,55 @@ export interface DropdownProps {
 
 const CardDetailsList: React.FC<CardDetailsListProps> = ({
   label,
   data,
-  addInput = false,
-  addButton = false,
-  dropdownTitles = [],
-  dropdownNextClick,
-  dropdownDeleteClick,
-  updatedItem,
-  addItem,
+  item,
 }) => {
-  const [valueInput, setValueInput] = useState<string>('');
-  const [valueInputUpdate, setValueInputUpdate] = useState<string>('');
-  const [isShow, setIsShow] = useState(false);
-  const [isEdit, setIsEdit] = useState(false);
-  const [selectedItem, setSelectedItem] = useState<DataProps>({
-    id: 0,
-    name: '',
-    status: '',
-  });
-
-  const showInput = () => {
-    setIsShow(true);
-  };
-
-  const handleSubmit = () => {
-    if (addItem) {
-      addItem({ id: new Date().getTime(), name: valueInput, status: 'todo' });
-    }
-  };
-
-  const handleSubmitUpdate = () => {
-    if (updatedItem) {
-      updatedItem({ ...selectedItem, name: valueInputUpdate });
-    }
-  };
-
-  const handleKeyPress = (e: KeyboardEvent<HTMLInputElement>) => {
-    if (e.key === 'Enter') {
-      handleSubmit();
-      setIsShow(false);
-      setValueInput('');
-    }
-  };
-  const handleKeyPressUpdate = (e: KeyboardEvent<HTMLInputElement>) => {
-    if (e.key === 'Enter') {
-      setValueInputUpdate('');
-      setIsEdit(false);
-      handleSubmitUpdate();
-    }
-  };
-
-  const dropdownStatusNext = (title: string | null | undefined) => {
-    let dropdownName = '';
-    const dropdownIndex = dropdownTitles.indexOf(title);
-
-    if (dropdownIndex !== -1 && dropdownIndex < dropdownTitles.length - 1) {
-      dropdownName = dropdownTitles[dropdownIndex + 1];
-    } else {
-      dropdownName = '';
-    }
+  const dispatch = useAppDispatch();
 
-    return dropdownName;
-  };
-
-  const titleFormatted = (title: string | null | undefined) => {
-    if (title) {
-      if (title.split('_')[1]) {
-        return title
-          .split('_')
-          .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
-          .join(' ');
-      } else {
-        return title.charAt(0).toUpperCase() + title.slice(1);
-      }
-    }
-  };
-
-  const getDropdowns = (item: DataProps) => {
-    let dropdownArr: any[] = [];
-
-    if (dropdownStatusNext(item.status)) {
-      dropdownArr.push({
-        name: titleFormatted(dropdownStatusNext(item.status)),
-        click: () =>
-          dropdownNextClick &&
-          dropdownNextClick(item, dropdownStatusNext(item.status)),
-      });
-    } else {
-      dropdownArr = [];
-    }
-
-    const updateAndRemove = [
-      {
-        name: 'Update',
-        click: () => {
-          setIsEdit(true);
-          setSelectedItem(item);
-          setValueInputUpdate(item.name);
-        },
-      },
-      {
-        name: 'Delete',
-        click: () => {
-          dropdownDeleteClick && dropdownDeleteClick(item);
-        },
-      },
-    ];
-
-    return [...dropdownArr, ...updateAndRemove];
+  const handleRemoveTitle = () => {
+    dispatch(setRemoveTrelloTitle(label));
```

Comment:
label unique mi? mesala label title "Todo" ve bir daha aynı "Todo" yazacak olursa 2 kez kaldırabilir problem değil mi?

sadece bir unique olmalı. id unique'yse id olur

Reply:
evet dediğin gibi id yaptım ki unique  olsun